### ISSUE TITLE
Not returning information on newly created record

### ISSUE THREAD

**Issue by perlgoodies** [rbw/pysnow#152]
I am new to Python (I come from a Perl background) and can't get the creation of the record to work.

In testing, the record IS being created but the return is not helpful. When I print result below, it's showing [Response - 201] instead of the sys_id or object of the record it actually created.

Am I using it wrong? My end goal is to get the sys_id of the new record.

`
# Define a resource, here we'll use the incident table API
incident = c.resource(api_path='/table/incident')

# Set the payload
new_record = {
    'short_description': 'Pysnow created incident',
    'description': 'This is awesome'
}

# Create a new incident record
result = incident.create(payload=new_record)
print(result)`

**Comment by rbw** [comment]
Hello,

Yes, that needs to be updated in the documentation.

To fetch the created record, use:
```python
result = incident.create(payload=new_record)
print(result.one())
```

Building something new? Can you use Python 3.7+ asyncio? If so, I'd take a look at [snow](https://github.com/rbw/snow) instead; a new library I'm working on, with a much improved API and documentation. It's currently in alpha (beta shortly) and I need testers. Should you find a bug, it'll fixed promptly.

**Comment by rbw** [comment]
Fixed in #153
